easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-1.7.1-foss-2017b.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'HarfBuzz'
+version = '1.7.1'
+
+homepage = 'http://www.freedesktop.org/wiki/Software/HarfBuzz'
+description = """HarfBuzz is an OpenType text shaping engine."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['http://www.freedesktop.org/software/harfbuzz/release/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['9645a6e83313b690602017f18d4eb2adf81f2e54c6fc4471e19331304965154e']
+
+dependencies = [
+    ('GLib', '2.53.5'),
+    ('cairo', '1.14.10'),
+    ('freetype', '2.8.1'),
```

Comment:
You'll need to stick to `freetype` 2.8 here, since that's already used as a dependency for `cairo`:

```
$ eb --from-pr 5388 --check-conflicts
== temporary log file in case of crash /tmp/eb-cb9Xcq/easybuild-uUlFW9.log

*** WARNING: Using easyconfigs from unstable (pending/failed tests or merge conflict) PR #5388 ***

Conflict found for dependencies of HarfBuzz-1.7.1-foss-2017b: freetype-2.8-GCCcore-6.4.0 vs freetype-2.8.1-GCCcore-6.4.0
        freetype-2.8-GCCcore-6.4.0 as dep of: GObject-Introspection-1.53.5-foss-2017b-Python-2.7.14, HarfBuzz-1.7.1-foss-2017b, X11-20171023-GCCcore-6.4.0, cairo-1.14.10-GCCcore-6.4.0, fontconfig-2.12.4-GCCcore-6.4.0
        freetype-2.8.1-GCCcore-6.4.0 as dep of: HarfBuzz-1.7.1-foss-2017b
Conflict between (dependencies of) easyconfigs: freetype-2.8-GCCcore-6.4.0 vs freetype-2.8.1-GCCcore-6.4.0
        freetype-2.8-GCCcore-6.4.0 as dep of: GObject-Introspection-1.53.5-foss-2017b-Python-2.7.14, HarfBuzz-1.7.1-foss-2017b, X11-20171023-GCCcore-6.4.0, cairo-1.14.10-GCCcore-6.4.0, fontconfig-2.12.4-GCCcore-6.4.0
        freetype-2.8.1-GCCcore-6.4.0 as dep of: HarfBuzz-1.7.1-foss-2017b
ERROR: One or more conflicts detected!
```